### ISSUE TITLE
Pylint highlight on column and not range; no hover message - VSCode

### ISSUE THREAD

**Issue by carlos-ventura** [PyCQA/pylint#5299]
### Bug description

When enabling pylint on the VSCode editor:
1 - The highlight of an error is not the whole word but just a column (just the first letter, image for clarification)
2 - There is no hover option to show the error when the cursor is on the highlight "letter"
Images below for better explanation
![bug](https://user-images.githubusercontent.com/71085933/141654540-4fd80ff1-fbba-46de-9ee7-944b49b2999c.png)
As one can see there is only  a small `~` below the the errors.
When the mouse is put on this `~` no message appears for a description of the error.

### Command used

```shell
Not relevant
```

### Pylint output

```shell
Unable to import 'networkx' pylint(import-error) [6, 1]
Unable to import 'bs4' pylint(import-error) [7, 1]
...
```

### Expected behavior

It is expected that the whole word of the error would be highlighted and when the mouse would be put on the top of the error (hover) a pop-up message would appear explaining the error.

### Pylint version

```shell
pylint 2.11.1
astroid 2.8.5
Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)]
```

### OS / Environment

Windows 10 Pro - 21H1

### Additional dependencies

VSCode version - 1.62.2

**Comment by cdce8p** [comment]
I've seen this too recently. Not sure the issue is `pylint`. We didn't change the output format. So I suspect it's something with the highlighting in either VSCode or the VSCode python extension.

Would be interesting to know the first affected version(s). Can't be too long ago, maybe `1.60` or `1.61`? Don't have the time to test this at the moment, unfortunately.

@carlos-ventura If you are able, I would suggest testing an old VSCode version and raising the issue in their repo.

**Comment by cdce8p** [comment]
This "might" be related: https://github.com/microsoft/vscode-python/issues/17398

**Comment by carlos-ventura** [comment]
@cdce8p I tried reverting versions starting from the most recent and the version 1.61.2 worked and pylint works as intended.
Another problem I am currently having is `previous run` only shows the result of the actual run, doesn't return the last one.
`(previous run: acutal_run_score, +0.00(always zero) )`

**Comment by cdce8p** [comment]
<img width="713" alt="Screen Shot 2021-11-14 at 22 41 57" src="https://user-images.githubusercontent.com/30130371/141700018-c3e7d66e-0344-4f4b-90b1-8c8daad02fc3.png">

An interesting observation: `refactor` messages don't seem to be effected only `error`.

**Comment by cdce8p** [comment]
@carlos-ventura The hover message has been fixed in `vscode`. I guess it will be included in `1.63.0` next month. https://github.com/microsoft/vscode/issues/136711

Just for reference, there are also open `vscode-python` issues for it:
https://github.com/microsoft/vscode-python/issues/17989
https://github.com/microsoft/vscode-python/issues/17882

On a related note, `pylint` should ideally output ranges for errors so IDEs can highlight them better. #5336
